modules/social_features/social_activity/modules/social_activity_filter/config/update/social_activity_filter_update_8002.yml

```diff
@@ -0,0 +1,402 @@
+views.view.activity_stream:
+  expected_config:
```

Comment:
Why is this necessary?

Reply:
I don't think we want to remove metadata for this right?

Reply:
This part was automatically generated by update_helper but I will try to clean up and check it

Reply:
I understand, unfortunately we can't always trust this :( it takes in to account your current active config. So also optional modules that were enabled/disabled for example.